python/rikai/vision.py

```diff
@@ -167,3 +171,49 @@ def to_numpy(self) -> np.ndarray:
             "bbox": self.bbox.to_numpy(),
             "score": self.score,
         }
+
+
+class VideoStream:
+    """VideoStream resource"""
+
+    __UDT__ = VideoStreamType()
+
+    def __init__(self, uri: str):
+        self.uri = uri
+
+    def __repr__(self) -> str:
+        return f"VideoStream(uri={self.uri})"
+
+    def _repr_html_(self):
+        """TODO: codec"""
+        from IPython.display import Video
+
+        if pathlib.Path(self.uri).exists():
+            path = pathlib.Path(self.uri).relative_to(os.getcwd())
+            return Video(path, width=480, height=320)._repr_html_()
```

Comment:
will the `width` and `height` be configurable ?

Reply:
we can add a rikai.options.display module that allows the user to configure this in the notebook. I was planning to do it in a separate PR